playerMat.scss

```diff
@@ -90,17 +90,22 @@
 	margin-right: 5px;
 }
 
+#floorContainer {
+	position: relative;
+}
+
 #floorLineTrashCan {
+	&:not(.hidden) {
+		display: flex;
+	}
 	display: none;
-	position: relative;
-	bottom: 78px;
+	position: absolute;
+	top: 2px;
 	left: 32px;
-	margin-bottom: -66px;
+	justify-content: center;
 }
 
 #tcText {
-	font-size: 72px;
-	position: relative;
-	bottom: 40px;
-	right: 6px;
+	font-size: 64px;
+	line-height: 66px;
```

Comment:
with line-height set to the height of the parent container, this text will always end up vertically centered, even if you change font-size.